### PR TITLE
ENH: Smaract Open Loop Scanning

### LINKED ISSUE #625: SmarAct Open Loop Motor Scanning
## Expected Behavior
We should be able to use SmarAct open loop motors in scans.

## Current Behavior
We currently cannot use SmarAct motors in scans, except perhaps using plan stubs like abs_set. 

## Possible Solution
Write a new class to do this. 

PR forthcoming. 

## Steps to Reproduce (for bugs)
Try to use a SmarActOpenLoop device in a scan. 

## Context
This will allow us to do scans of SmarAct tip-tilt mirrors for beam pointing, and could be used for other open loop motors such as irises. 

## Your Environment
Laspython.

### PR DESCRIPTION
**Pull request by slactjohnson**
## Description
Adds a new class, SmarActOpenLoopPositioner for doing open loop motor scanning with SmarAct stages. 

## Motivation and Context
This allows us to scan SmarAct stages that are open loop, such as tip-tilt stages. This will be used for doing laser beam pointing/centering. 

Closes #625 

## How Has This Been Tested?
Tested on a stage in the Heinz lab. 

## Where Has This Been Documented?
The code and docstrings. 

## Pre-merge checklist
- [X] Code works interactively
- [X] Code contains descriptive docstrings, including context and API
- [X] New/changed functions and methods are covered in the test suite where possible
- [X] Test suite passes locally
- [X] Test suite passes on travis
- [X] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [X] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/epics_motor.py [hunk 1]
```diff
@@ -531,15 +533,37 @@ class SmarActOpenLoop(Device):
...
+    setpoint = Cpt(EpicsSignal, ':SET_TOTAL_STEP_COUNT')
+    readback = Cpt(EpicsSignalRO, ':TOTAL_STEP_COUNT')
+    atol = 1  # step count after move should be exact, but let's be cautious
+
+    egu = 'steps'
+
+    open_loop = Cpt(SmarActOpenLoop, '', kind='normal')
+
+    def done_comparator(self, readback, setpoint):
+        if setpoint-self.atol < readback and readback < setpoint+self.atol:
```
Comment (klauer) [inline pcdsdevices/epics_motor.py:561]:
```suggestion
        return setpoint-self.atol < readback < setpoint+self.atol
```

File: docs/source/upcoming_release_notes/625-SmarAct-open-loop-scanning.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- SmarActOpenLoopPositioner: Class intended for performing Bluesky scans using
+                             open loop SmarAct motors.
```
Comment (klauer) [inline docs/source/upcoming_release_notes/625-SmarAct-open-loop-scanning.rst:19]:
```suggestion
   open-loop SmarAct motors. 
```

**Review by klauer — APPROVED** [review]
Couple minor things - one just a simplification. All seems good, though I wonder if there's might be a benefit in merging `SmarActOpenLoop` and `SmarActOpenLoopPositioner` at some point.

File: pcdsdevices/epics_motor.py (hunk 1 above)
Comment (slactjohnson) [reply pcdsdevices/epics_motor.py:561]: @klauer Yes, I agree.

**Pull request merged**